Composer/.eslintrc.js

```diff
@@ -105,5 +117,11 @@ module.exports = {
         'security/detect-unsafe-regex': 'off',
       },
     },
+    {
+      files: ['__tests__/**/*'],
+      rules: {
+        'node/no-missing-import': 'off'
+      }
+    }
```

Comment:
This should move to the block above. Already takes care of test overrides.

Reply:
merged above